src/sign/mod.rs

```diff
@@ -1,10 +1,57 @@
 //! DNSSEC signing.
 //!
 //! **This module is experimental and likely to change significantly.**
+//!
+//! Signatures are at the heart of DNSSEC -- they confirm the authenticity of a
+//! DNS record served by a secure-aware name server.  But name servers are not
```

Comment:
RFC 9499 defines "DNSSEC-aware" and refers to "security-aware" but doesn't mention "secure-aware".

Reply:
Oops, typo.  I meant security-aware.